odbc/src/api/definitions.rs

```diff
@@ -153,43 +153,48 @@ pub enum EnvironmentAttribute {
 #[repr(i32)]
 #[derive(Debug, PartialEq, Eq, Clone, Copy, FromPrimitive)]
 pub enum StatementAttribute {
-    SQL_ATTR_APP_ROW_DESC = 10010,
-    SQL_ATTR_APP_PARAM_DESC = 10011,
-    SQL_ATTR_IMP_ROW_DESC = 10012,
-    SQL_ATTR_IMP_PARAM_DESC = 10013,
     SQL_ATTR_CURSOR_SCROLLABLE = -1,
     SQL_ATTR_CURSOR_SENSITIVITY = -2,
+    SQL_ATTR_QUERY_TIMEOUT = 0,
+    SQL_ATTR_MAX_ROWS = 1,
+    SQL_ATTR_NOSCAN = 2,
+    SQL_ATTR_MAX_LENGTH = 3,
     SQL_ATTR_ASYNC_ENABLE = 4,
-    SQL_ATTR_CONCURRENCY = 7,
+    SQL_ATTR_ROW_BIND_TYPE = 5,
     SQL_ATTR_CURSOR_TYPE = 6,
+    SQL_ATTR_CONCURRENCY = 7,
+    SQL_ATTR_KEYSET_SIZE = 8,
+    // Never renamed to SQL_ATTR_ROWSET_SIZE
+    SQL_ROWSET_SIZE = 9,
```

Comment:
It's unfortunate, but I don't think we should change the name. My preference is to stick with the (weird) conventions from the header files.

Reply:
agreed